docs/core/metrics.md

```diff
@@ -98,6 +104,28 @@ You can create metrics using `add_metric`, and you can create dimensions for all
 !!! note "Metrics overflow"
 	CloudWatch EMF supports a max of 100 metrics per batch. Metrics utility will flush all metrics when adding the 100th metric. Subsequent metrics, e.g. 101th, will be aggregated into a new EMF object, for your convenience.
 
+
+### Adding default dimensions
+
+You can add default metric dimensions to ensure they are persisted across Lambda invocations using `set_default_dimenions`.
+
+!!! info "If you'd like to remove them at some point, you can use `clear_default_dimensions` method"
+	Note that they continue to count against the maximum of 9 dimensions.
```

Comment:
This sounds like even if you clear them, they still count. Is that the case?

Reply:
If you clear them, nope. The limit counts when we serialize and dump the JSON to stdout. If they are cleared, there will be no default dimensions in the EMF blob